students/jongho/users/views.py

```diff
@@ -1,3 +1,38 @@
-from django.shortcuts import render
+import json,re
 
-# Create your views here.
+from django.views import View
+from django.http  import JsonResponse
+from django.db    import DataError
+
+from .models      import User
+
+class SignUp(View):
```

Comment:
View 이름은 컨벤션에 맞게 항상 아래와 같이 작성해주세요! 그에 맞게 urls.py도 수정해주세요!
```suggestion
class SignUpView(View):
```